packages/browser/src/core/buffer/snippet.ts

```diff
@@ -32,8 +36,7 @@ type SnippetBuffer = SnippetWindowBufferedMethodCall[]
  */
 export const getSnippetWindowBuffer = (): PreInitMethodCall[] => {
```

Comment:
Now that the behaviour has been changed, it's more like `popSnippetWindowBuffer`, similar to `git stash pop` and `Array.prototype.pop`. Usually `get*` functions are non-mutating.

Just a thought, totally not necessary.